src/styles/tailwind.css

```diff
@@ -0,0 +1,19 @@
+.m-0 {
+  margin: 0;
+}
+
+.mt-0 {
+  margin-top: 0;
+}
+
+.mr-0 {
+  margin-right: 0;
+}
+
+.mb-0 {
+  margin-bottom: 0;
+}
+
```

Comment:
[Seems bootstrap already supports it.](https://getbootstrap.com/docs/4.6/utilities/spacing/#examples)
Since we are using bootstrap already, I think we should follow the rule of thumb of `not introducing new frameworks/utils that serves the same purpose`. ref [discussion with rick ](https://github.com/oursky/mockuphone.com/pull/26#discussion_r1619103766)